tests/components/test_kubernetes_app.py

```diff
@@ -33,6 +33,7 @@ def config(self) -> PipelineConfig:
             defaults_path=DEFAULTS_PATH,
             environment="development",
             helm_diff_config=HelmDiffConfig(enable=True),
+            broker="",  # This is missing
```

Comment:
@raminqaf @disrupted PyCharm was complaining that broker is not set. Field default is `...`. Do you know why it works in CI? Can also be because I am on Windows and there are problems with environment variable config

Reply:
a lot of code analysis tools have problems with Pydantic, especially with off-spec stuff that works differently to the `dataclass_transforms` stuff.

Usually the workaround is to change the definition from `Field(...)` to `Field(default=...)` but I think that's already the case here.

Reply:
It's not code analysis but when I run Pytest via PyCharm it just aborts

Reply:
ah, I don't know then.

Reply:
But as far as I understand, broker is a required Field. Obviously it was not set in this constructor. How is it then set in tests? Is there an environment variable defined somewhere?

Reply:
`default =...` means it is mandatory. But I'd like to know how this was not broken before. 😅 
I think the value is read from the `config.yaml` file in the root

Reply:
From here:
https://github.com/bakdata/kpops/blob/main/config.yaml#L2

Reply:
Ah then it is just a problem with the base dir in my PyCharm